_wiki/FAQ/Windows.md

```diff
@@ -3,188 +3,146 @@ title: Windows FAQ
+Follow the [instructions](https://learn.microsoft.com/en-us/mem/intune/user-help/you-need-to-enable-uac-windows#adjust-protection-level) provided by Windows and adjust protection level to at least:
 
-Starting OpenTabletDriver minimised can also be done by changing the properties of a shortcut.
-To do this, right click the OpenTabletDriver shortcut, go to properties > run > minimised.
+- Notify me when apps try to make changes to my computer.
 
----
+Then restart your computer to apply.
+
+If you are still getting the warning, make sure that you are not using the account called `Administrator`. This is *NEVER* meant to be used as a normal account for daily use. Create a new normal account with administrator privileges and use that instead.
 
-### Plugin Manager won't open/I get a warning on startup? {#administrator}
+> We are aware that most custom ISOs use the Administrator account. Due to the nature of custom ISOs, we *do not* provide official support to them.
 
-When Starting OpenTabletDriver you may get an error warning you about Administrator Permissions.
-OpenTabletDriver should NEVER be run as Administrator or with Administrator permissions.
-This causes the plugin manager, FAQ page, and Tablet debugger to not work, and can interfere with several other features.
+If you need to interact with a program that runs with Administrator privileges, use [VMultiMode](https://github.com/X9VoiD/VoiDPlugins/wiki/VMultiMode).
 
-The following instructions should fix your issue under normal circumstances, but if you're using the (disabled in a fresh Windows install by default) account called "Administrator",
-usually because you're using a "custom" iso (unofficial version of Windows, also sometimes referred to as a "debloated Windows") install,
-these instructions will not work for you.
+---
 
-We generally recommend against using altered or 3rd party versions of Windows,
-as OTD is likely to not be the only software adversely affected by the changes these custom ISOs make.
+### Plugin manager is not showing up
 
- - Press the Windows key and search` User Account Control`.
- - Move the slider to one of the top 2 options.
- - Click OK, then restart your computer.
+It is likely because you are running OpenTabletDriver with the Administrator account. See [above](#administrator).
 
 ---
 
-### Why is my interface completely blank? {#rtss}
+### OpenTabletDriver is completely blank {#rtss}
 
-If you are experiencing random crashes with no warning, or the interface is completely blank and/or not accepting input, and you have RiverTuner Statistics Server
-installed, it may be hooking into OpenTabletDriver and preventing the UX from functioning correctly.
+If you are experiencing random crashes with no warning, or the interface is completely blank and/or not accepting input, and you have RiverTuner Statistics Server installed, it may be hooking into OpenTabletDriver and prevents OpenTabletDriver.UX.Wpf from functioning correctly.
```

Comment:
typo:
```suggestion
If you are experiencing random crashes with no warning, or the interface is completely blank and/or not accepting input, and you have RivaTuner Statistics Server installed, it may be hooking into OpenTabletDriver and prevents OpenTabletDriver.UX.Wpf from functioning correctly.
```